lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -62,11 +61,12 @@ class CourseCardModel extends Backbone.Model {
   }
 
   getEnrollableCourseRuns() {
-    const rawCourseRuns = _.where(this.context.course_runs, {
-      is_enrollment_open: true,
-      is_enrolled: false,
-      is_course_ended: false,
-      status: 'published',
+    // eslint-disable-next-line arrow-body-style
```

Comment:
I think the below line can be written without the parens and return statement, you'd get to remove the linting disable.

```javascript
const rawCourseRuns = this.context.course_runs.filter( run => 
  run.is_enrollment_open &&
  !run.is_enrolled &&
  !run.is_course_ended &&
  run.status === 'published';
);
```

Reply:
That doesn't work because eslint complains about the terse style being used over multiple lines. Braces don't help, because then it wants the full style. But I found a way! You have to use parens to use a terse style over multiple lines. Like so:

```
    const rawCourseRuns = this.context.course_runs.filter(run => (
      run.is_enrollment_open &&
      !run.is_enrolled &&
      !run.is_course_ended &&
      run.status === 'published'
    ));
```

So that's what I'll do. Thanks for inspiring a second look here.